crates/runtime/benches/queries/README.md

```diff
@@ -1,10 +1,11 @@
 # Spice.ai OSS Benchmarks
 
-## TPC-DS (Decision Support Benchmark)
+## TPC-DS (Decision Support Benchmark) Limitations
 
-### Intervals like 'date + 30 days' are not supported
+### Interval Arithmetic (date + 30 days) Not Supported
```

Comment:
I was under impression that there was slightly different case/query and error (`d3.d_date > d1.d_date + 5`). I would keep them separately:

Intervals like 'date + 30 days' or `date + 5` are not supported

Reply:
Pls also add example to this section

```
# fail
SELECT (now() + 30 days);
# fail
SELECT (now() + 30);

# success
SELECT (now() + INTERVAL '30 days');
```